docs/learn-staking.md

```diff
@@ -74,74 +78,72 @@ There are three different accounts for managing your funds: `Stash`, `Controller
 
-Since validator slots will be limited, most of those who wish to stake their DOTs and contribute economic security to the network will be nominators. Validators do most of the heavy lifting, they produce new block candidates in BABE, vote and come to consensus in GRANDPA, validate STF of parachains, and possibly some other responsibilities in regard to data availability. Nominators, on the other hand do not need to do anything once they have bonded their DOTs. The experience of the nominator is similar to "set it and forget it" while the validator will be doing an actual service for the network by performing the critical operations. For this reason, the validator has certain privileges in regard to the payout of the staking mechanism and will be able to declare its own allocation before the share is divided to nominators.
+Since validator slots will be limited, most of those who wish to stake their DOTs and contribute economic security to the network will be nominators. Validators do most of the heavy lifting, they produce new block candidates in BABE, vote and come to consensus in GRANDPA, validate STF of parachains, and possibly some other responsibilities in regard to data availability. Nominators, on the other hand, do not need to do anything once they have bonded their DOTs. The experience of the nominator is similar to "set it and forget it" while the validator will be doing an actual service for the network by performing the critical operations. For this reason, the validator has certain privileges in regard to the payout of the staking mechanism and will be able to declare its own allocation before the share is divided to nominators.
 
 ![staking](assets/NPoS/article-2.png)
 
 ### Want to stake DOTs?
 
-- [Nominator Guide](maintain-guides-how-to-nominate) - Become a nominator on the Alexander testnet.
-- [Validator Guide](maintain-guides-how-to-validate) - Become a validator on the Alexander testnet.
+* [Nominator Guide](maintain-guides-how-to-nominate) - Become a nominator on the Alexander testnet.
+* [Validator Guide](maintain-guides-how-to-validate) - Become a validator on the Alexander testnet.
 
 ## Slashing
 
-Slashing will happen if a validator misbehaves(e.g. always offline, attack the network or running modified software) in the network, they and their nominators will get slashed by losing a percentage of their bonded/staked DOTs. 
+Slashing will happen if a validator misbehaves (e.g. always offline, attack the network or running modified software) in the network. They and their nominators will get slashed by losing a percentage of their bonded/staked DOTs.
 
-As validators have more DOTs staked, they will get slashed more, so we encourage nominators to shift their nominations to less popular validators to reduce the risk of being lost more. 
+As validators have more DOTs staked, they will get slashed more, so we encourage nominators to shift their nominations to less popular validators to reduce the possible losses.
```

Comment:
This really only applies if you're a huge nominator since slashes are pro-rata. If you are a huge nominator then you should divide your nomination amongst more validators.

Also don't like this "As validators have more DOTs staked" part because it could be read as "Validators have more DOTs than nominators".

Reply:
Agreed, it's confusing as a single sentence.. what is meant is "validator pools with larger total stake backing them will get slashed more than less popular ones, so nominators should shift nominations to the lowest stake val pool they trust"

Reply:
The idea is to discourage oversubscribing on a validator from nominators

Reply:
```suggestion
Validator pools with larger total stake backing them will get slashed more harshly than less popular ones, so we encourage nominators to shift their nominations to less popular validators to reduce the possible losses.
```